src/libraries/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -91,6 +91,7 @@ System.Text.CodePagesEncodingProvider</PackageDescription>
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
+    <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
```

Comment:
@MichaelSimons @ViktorHofer this package reference is causing the error in CI about prebuilts.

I tried adding the package to source-build-reference-packages but I get this error:

```
 $ ./generate.sh --package System.ValueTuple,4.5.0
...
Build succeeded in 4.6s
Restore complete (1.9s)
  PackageSourceGeneratorTask succeeded (2.9s) → artifacts/bin/PackageSourceGeneratorTask/Debug/net10.0/PackageSourceGeneratorTask.dll
  PackageSourceGenerator succeeded (0.4s) → Package source generation skipped for System.ValueTuple (v4.5.0) as it doesn't contain any compile items.
```

What options do we have for such a case where no compile items are found in a published package, but adding such package as reference causes prebuilt errors?

Reply:
Replied offline already. The System.ValueTuple package is a netfx support package that doesn't need to be referenced on NET Standard.

Make sure that the package ref is conditioned to TargetFrameworkIdentifier == NETFramework.